### PR TITLE
Update datasources.tsv

### PR DESCRIPTION
**Pull request by hbasaric**
Adding three new datasources: github, cell ontology, evidence ontology

### CODE REVIEW

File: datasources.tsv [hunk 1]
```diff
@@ -139,3 +139,6 @@ Wikipedia	Wi	http://www.wikipedia.org	https://en.wikipedia.org/wiki/$id	Acetate
 WormBase	W	http://www.wormbase.org/	http://www.wormbase.org/db/gene/gene?name=$id;class=Gene	WBGene00000001	gene	Caenorhabditis elegans	1	urn:miriam:wb	^WBGene\d{8}$	WormBase	P3860	wormbase	gene
 XMetDB	Xm	http://www.xmetdb.org/	http://www.xmetdb.org/xmetdb/protocol/$id	XMETDB171	interaction		1		^XMETDB\d+$	Xenobiotics Metabolism Database		xmetdb	interaction
 ZFIN	Z	http://zfin.org	http://zfin.org/action/marker/view/$id	ZDB-GENE-041118-11	gene	Danio rerio	1	urn:miriam:zfin	ZDB\-GENE\-\d+\-\d+	ZFIN Gene	P3870	zfin	gene
+Github	github	https://github.com	https://github.com/$id	githubuser	author		0	urn:miriam:github	^[a-zA-Z0-9-_]+(/[a-zA-Z0-9-_]+)?$	Github		github	author
```
Comment (egonw) [inline datasources.tsv:142]: Plz change to GitHub in the first column and in the later column near the end.

Comment (egonw) [reply datasources.tsv:142]: In the example identifier, use something non-random, plz. Who knows what may show up there in the future.

**Review by egonw — CHANGES_REQUESTED** [review]
Also, are you okay if a squash the commit later? Since you did not use non-standard commit messages, that sounds a logical approach to me.

File: datasources.tsv (hunk 1 above)
Comment (hbasaric) [reply datasources.tsv]: Thank you, made the changes and squashed the commits into 1 commit.

**Review by egonw — CHANGES_REQUESTED** [review]
Please change the example `CL:0000062` to `0000062`.

**Review by egonw — APPROVED** [review]

**Pull request merged**